pkgs/tools/security/spectre-cli/default.nix

```
@@ -0,0 +1,67 @@
+{ lib
+, gcc11Stdenv
+, fetchFromGitLab
+, cmake
+, libsodium
+, json_c
+, ncurses
+, libxml2
+, jq
+, ...
+}:
+
+gcc11Stdenv.mkDerivation {
```

```suggestion
stdenv.mkDerivation {
```
GCC 11 is the default for {x86_64,aarch64}-{darwin,linux}.